aibolit/scripts/02-calculate-metrics.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+if not os.path.isdir('02'):
+    os.makedirs('02')
+f = open('02/metrics.csv', 'w+')
+f.close()
+f1 = open("01/found-java-files.txt", "r")
```

Comment:
@zuoqin let's give this one a more descriptive name too. And again let's use `with`:

https://www.pythonforbeginners.com/files/with-statement-in-python

Reply:
@carlosmiranda, 01/found-java-files.txt file name comes from previous script and is defined by @yegor256